environments/azure/flyte-core/values-aks.yaml

```diff
@@ -242,7 +242,11 @@ configmap:
           - azure.workload.identity/use: "true"
         default-env-vars:
           - AZURE_STORAGE_ACCOUNT_NAME: "{{ .Values.userSettings.storageAccountName }}"
-        default-pod-template-name: "flyte-template" 
+  # Uncomment this line if you plan to request specific accelerator models
```

Comment:
Could we not have `gpu-device-node-label` & `gpu-partition-size-node-label` enabled by default? This way we dont would need to adjust the flyte helm chart on this chapter. In terms of the learning purpose of this chapter we could still link these 2 entries. Thoughts?

Reply:
We could but it would apply a nodeSelector that could interfere with scheduling, especially if users are not planning to request named devices or partitions